src/main/java/org/eclipse/yasson/internal/serializer/JsonObjectSerializer.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2016, 2017 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Order of imports is not enough (and shall not be done) to update copyright year change. I would advise revert here.

Reply:
This change may look like being not needed. But We are going to integrate checkstyle plugin in near future and and modified imports are matching new rules so it still make sense to keep this change here.